WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -114,7 +114,7 @@ final class PostListViewController: AbstractPostListViewController, UIViewContro
         }
         let updatedIndexPaths = (tableView.indexPathsForVisibleRows ?? []).filter {
             let post = fetchResultsController.object(at: $0)
-            return updatedObjects.contains(post)
+            return updatedObjects.contains(post) || updatedObjects.contains(post.original())
```

Comment:
The height no longer changes, so we should now move the notification observer to the cell itself, which will be safer